### PR TITLE
More APIs - Round 4

### PR DESCRIPTION
**Pull request by shadinaif**
**v0.4.5**
* feat: add rate limit to accessible-info API

**v0.4.3**
* feat: get accessible tenant API
* feat: add hidden_courses to statistics API
* fix: remove author_name from courses API
* feat: new API learners-course

### CODE REVIEW

File: futurex_openedx_extensions/dashboard/views.py [hunk 1]
```diff
@@ -219,3 +226,24 @@ def get(self, request, *args, **kwargs):  # pylint: disable=no-self-use
         return JsonResponse({
             'version': futurex_openedx_extensions.__version__,
         })
+
+
+class AccessibleTenantsInfoView(APIView):
+    """View to get the list of accessible tenants"""
+    permission_classes = []
```
Comment (OmarIthawi) [inline futurex_openedx_extensions/dashboard/views.py:233]: Please add `Token` authentication and limit to superusers.

**Review by OmarIthawi — CHANGES_REQUESTED** [review]
Looks great, thanks @shadinaif!

File: futurex_openedx_extensions/dashboard/views.py (hunk 1 above)
Comment (shadinaif) [reply futurex_openedx_extensions/dashboard/views.py:239]: will do 👍🏼

**Review by OmarIthawi — APPROVED** [review]
@shadinaif Looks good to me once the `Token` authentication is added to the tenant info API.

File: futurex_openedx_extensions/dashboard/views.py (hunk 1 above)
Comment (shadinaif) [reply futurex_openedx_extensions/dashboard/views.py]: @OmarIthawi , I added a rate limit for anonymous access. The API is accessible as anonymous or system-staff. No rate limit on system-staff users. Do we still need an API Key or Token? I guess not

**Comment by shadinaif** [comment]
API key to be added later

**Pull request merged**